test/org/zaproxy/zap/extension/websocket/utility/HttpHandshakeUnitTest.java

```diff
@@ -0,0 +1,25 @@
+package org.zaproxy.zap.extension.websocket.utility;
+
+import org.apache.commons.httpclient.URI;
+import org.junit.Test;
+import org.zaproxy.zap.extension.websocket.ExtensionWebSocket;
+import org.zaproxy.zap.extension.websocket.client.HttpHandshake;
+import org.zaproxy.zap.extension.websocket.client.HttpHandshakeRequestHeader;
+import org.zaproxy.zap.extension.websocket.client.HandshakeSender;
+
+public class HttpHandshakeUnitTest {
+    
+    @Test
```

Comment:
You could do something like: `@Test(expected = Exception.class)`

Also the function name is missing a "w".